config/filter.d/scanlogd.conf

```diff
@@ -0,0 +1,9 @@
+# Fail2Ban filter for port scans detected by scanlogd
+
+[Definition]
+
+failregex = scanlogd:\ <HOST>\ to\ .*\ ports\ .*
```

Comment:
this RE is vulnerable due to catch-all (`.*`) and missing anchors; we don't accept filters with unjustified catch-all's anymore

Reply:
@sebres: ack, PR amended. Please revisit.

Reply:
There are still no tests in fail2ban/tests/files/logs/scanlogd, so test suite fails in CI.
Either add the tests or provide a log excerpt with messages the filter match (then I'll add it to the PR by myself).